### PR TITLE
[+] Command generate - mongo hasMany

### PR DESCRIPTION
**Pull request by slimee**

### CODE REVIEW

**Comment by slimee** [comment]
[Lumber - Generate command creates hasMany relationships on NoSQL databases](https://trello.com/c/upSw1D9C/3871-lumber-generate-command-creates-hasmany-relationships-on-nosql-databases)

File: services/analyze-mongo-collections.js [hunk 1]
```diff
@@ -14,11 +15,22 @@ const mapReduceOptions = {
   limit: 100,
 };
 
-// eslint-disable no-undef
+/* eslint-disable */
```
Comment (DrRaider) [inline services/analyze-mongo-collections.js]: Why disable the linter here ?
Replace `var` by `let` or `const`, and use `Object.{keys, values, entries}` and iterate over the resulting array as suggested.

Comment (slimee) [reply services/analyze-mongo-collections.js]: In this file, `mapCollection` and `reduceCollection` are not conventionnal functions. This two pieces of code run on the mongoDB side.
This is done this the use of the mongoDB `mapReduce` function.

This is the reason why we have `var` and other old school things: mongoDB is not compatible with last JS versions.

Comment (DrRaider) [reply services/analyze-mongo-collections.js]: Maybe just disable the specific linter rules for this block of code, if we modify it later, we might miss other rules if we disable everything ;)

Comment (slimee) [reply services/analyze-mongo-collections.js]: good idea!

**Pull request merged**